README.md

```diff
@@ -1,27 +1,62 @@
+**Design Pattern:** - Following the CQRS pattern (Command and Query Responsibility Segregation)
+**Naming convention, file organization, and Comments:** 
+- Use camel case while declaring method parameters, property creation, and defining the model.
+- Static files or logged files should be maintained in separate folders.
+- Only include comments when necessary to provide clarification or context for complex logic, prioritize self-explanatory code when possible.
+### Development Workflow:
+#### Ticket Management:- 
+Agile methodology is followed, with the team working during a 2-week sprint.
+- Sprint planning sessions determine the sprint size, included tickets, and priority items.
+- Prior to sprint planning, tickets are discussed, refined, and scored during a refinement session.
+- After a ticket is assigned in JIRA, developers review the ticket details, including requirements, acceptance criteria, and any relevant documentation.
+- Once the scope and understanding are clear, developers proceed to work on the ticket.
+#### Version Control:-
+ Developers utilize Git for version control to manage changes to the codebase.
+ - Before starting work on a ticket, developers create a new branch from the develop branch, usually following a naming convention that includes the ticket number and a brief description. (Use all lower case for your branch names to avoid casing issues)
+ - Throughout the development process, developers commit their changes to the local branch, providing meaningful commit messages to track progress and changes.
+ - Periodically, developers push their local branches to the remote repository (e.g., GitHub) to collaborate with other team members and ensure backup of their work.
+ - Once the ticket is completed and ready for review, developers create a Pull Request (PR) to merge their changes into the develop branch, triggering the code review process.
+#### Code Review Process:-
+ Once the developer considers the task completed, they create a pull Request (PR) to merge their changes into the develop branch.
+ - Other developers are assigned to review the PR.- Reviewers provide feedback through comments or during a call.
+ - After addressing any feedback and ensuring the code meets the projects standards, the PR is approved.
+ - Upon approval, the changes are merged into the develop branch.
+#### Deployment Process:- 
+- After merging into the develop branch, changes are deployed to the test environment for further testing.
+- Upon successful testing, changes are deployed to the production environment.
+### Setup Instructions:-
+https://github.com/bcgov/moh-pidp/tree/develop/workspace#readme
+https://github.com/bcgov/moh-pidp/blob/develop/backend/webapi/README.md
+
+### My Onboarding Process:- 
```

Comment:
Would probably be best to have a comprehensive set of instructions on how to setup the project in this README. For example the backend README does not contain information on what to install (.NET, Entity Framework etc). Could you provide from your experience what you did to setup the project?